media_kit_test/lib/common/widgets.dart

```diff
@@ -183,7 +183,7 @@ class _SeekBarState extends State<SeekBar> {
       children: [
         const SizedBox(width: 48.0),
         IconButton(
-          onPressed: widget.player.playOrPause,
+          onPressed: playing ? widget.player.pause : widget.player.play,
```

Comment:
Why?

Reply:
To avoid confusion while testing, since using playorpause can do the same function